list-targets-with-changed-files/src/index.ts

```diff
@@ -157,7 +157,10 @@ try {
   }
 
   core.setOutput('tfmigrate_targets', tfmigrateObjs);
-  core.setOutput('terraform_targets', terraformTargetObjs);
+  if (tfmigrateObjs.length == 0) {
+    // Running terraform when tfmigrate has nothing to do
+    core.setOutput('terraform_targets', terraformTargetObjs);
+  }
```

Comment:
The differences other than these changes are the result of running `npm run build`.